crates/holo_hash/Cargo.toml

```diff
@@ -37,14 +37,13 @@ serde_json = { version = "1.0.51", features = [ "preserve_order" ] }
 [features]
 default = ["serialization"]
 full = [
-    "fixturators",
     "hashing",
     "encoding",
     "sqlite"
 ]
 
 fixturators = ["fixt", "rand", "hashing", "encoding"]
-hashing = ["futures", "must_future", "blake2b_simd", "serialization"]
+hashing = ["futures", "must_future", "blake2b_simd", "serialization", "encoding"]
```

Comment:
`hashing` internally depends on `encoding`, this detail doesn't need to be exposed to consumers of the library